meganote-refactor/app/content/app.css

```diff
@@ -0,0 +1,125 @@
+body {
+  font-family: Oxygen, Helvetica, Arial, sans-serif; }
+
+h1, h2, h3, h4, h5 {
+font-family: Merriweather;
+font-weight: 300;
+color: #FE5F55; }
+
+.container {
+  width: 100% !important;
+  padding-left: 0 !important;
+  padding-right: 0 !important; }
+
+.row {
+  margin-left: 0 !important;
+  margin-right: 0 !important; }
+
+.col-xs-12 {
+  padding-left: 0 !important;
+  padding-right: 0 !important; }
+
```

Comment:
We should try to avoid using `!important` as much as we can. `!important` overrides how styles apply so if we want to use `.container`, `.row`, or `.col-xs-12` later on we won't be able to override these styles specifically.

Alternatively, we would try to scope the styles a little further. Something like this should work:
```
.container.wrap {
  width: 100%;
  padding-left: 0;
  padding-right: 0;
}

.container .row {
  margin-left: 0;
  margin-right: 0;
}

.row .col-xs-12 {
  padding-left: 0;
  padding-right: 0;
}
```